builder/llpcBuilderImplInOut.cpp

```diff
@@ -556,8 +556,10 @@ Value* BuilderImplInOut::EvalIJOffsetSmooth(
     // Extract <I,J> part of that, and divide by W.
```

Comment:
This comment seems wrong.  It is saying you want to divide by `W`, which is what the old code did.  The new code is multiplying by `W`.

Reply:
Thanks you for pointing out the issue. I have create a PR to revise the comment and related variable naming to correctly reflect the code behavior. https://github.com/GPUOpen-Drivers/llpc/pull/427